assets/reader-activation/auth.scss

```diff
@@ -140,51 +173,70 @@
 		&__wrapper {
 			width: 100%;
 			max-width: 544px;
-			background: #fff;
+			background: white;
 			position: relative;
 			max-height: 100vh;
 			overflow-y: auto;
 
+			transition: height 125ms ease-in-out;
+
 			.newspack-reader__auth-form__inline & {
 				margin: auto;
 				max-width: 780px;
 			}
 		}
 
 		&__header {
+			align-items: baseline;
 			display: flex;
+			flex-wrap: wrap;
+			gap: 0.5em;
 			justify-content: space-between;
-			align-items: center;
+			margin: 0 0 1em;
+			padding-right: 40px;
+
+			@media screen and ( min-width: 600px ) {
+				padding-right: 0;
+			}
+
 			h2 {
-				font-size: 1.3em;
+				font-size: 1em;
+				line-height: 1.5;
+				margin: 0;
+
+				@media screen and ( min-width: 782px ) {
+					font-size: 1.3125em;
+					line-height: 1.523809;
+				}
 			}
 			a {
-				font-size: 0.8em;
-				color: #36f;
-				margin-left: 1rem;
-			}
-			@media screen and ( max-width: 744px ) {
-				justify-content: flex-start;
+				font-size: 0.8125em;
+				line-height: 1.2307692308;
 			}
 		}
 
 		&__content {
-			padding: 64px;
+			padding: 16px;
+
+			@media screen and ( min-width: 600px ) {
+				padding: 32px;
+			}
 
-			@media screen and ( max-width: 744px ) {
-				padding: 0 5vw 5vw 5vw;
+			@media screen and ( min-width: 782px ) {
+				padding: 64px;
 			}
+
 			form {
 				transition: opacity 0.2s ease-in-out;
+
+				> *:last-child {
+					margin-bottom: 0 !important;
+				}
 			}
 			input[type='email'],
 			input[type='password'] {
 				width: 100%;
 			}
-			[type='submit'] {
```

Comment:
Was this intentionally removed? This makes the submit button take on the theme's secondary color, which by default is quite a bit less prominent than the primary.

With primary color (`master`):
<img width="820" alt="Screen Shot 2022-08-08 at 9 22 49 PM" src="https://user-images.githubusercontent.com/2230142/183557669-9d83d406-b3f0-4fdd-be8f-77e5bc00a298.png">

With secondary color (this PR):
<img width="811" alt="Screen Shot 2022-08-08 at 9 23 10 PM" src="https://user-images.githubusercontent.com/2230142/183557634-23be4b0f-0cb0-489c-bc71-e27b4b6c3831.png">

Reply:
Yes, it's intentional. I wanted to match the theme's styles.
Also for now, we said we would use custom CSS to apply a red colour #DD3333 to these buttons.
That being said, I'd like to introduce a setting somewhere (Site Design? Reader Activation?) to control the background/text colour for them.